Json exercise (#3)
ememmr/sky-training: 0 - Starting point - Mandatory/README.md

# Session 0 - Domain Modeling / Traits
*Friday 27 Jan*

## Pre-readings
https://docs.scala-lang.org/scala3/book/domain-modeling-tools.html
https://docs.scala-lang.org/scala3/book/domain-modeling-fp.html#modeling-the-data

> https://www.baeldung.com/scala/traits

Reviewer: oops the markdown formatting comes out with all the links on one line. You can add bullets or put a new line after each one.

## Class time

> Create a giter8 template project: https://github.com/scala/scala-seed.g8.
> In this project create a basic implementation of what we have in Ingestion Factory with a Feed that has an Origin and Destination trait defined.
> We can answer any questions you have on the readings and you can talk us through the code that you've written. We could also start to think about how we would implement reading from a source file using the data models you've created in your projects.

Reviewer: ```suggestion
Create a giter8 template project: https://github.com/scala/scala-seed.g8.
In this project create a basic implementation of what we have in Ingestion Factory with a Feed that has an Origin and Destination trait defined.
```

> ## Class time
> Create a giter8 template project: https://github.com/scala/scala-seed.g8.
> In this project create a basic implementation of what we have in Ingestion Factory with a Feed that has an Origin and Destination trait defined.
> We can answer any questions you have on the readings and you can talk us through the code that you've written. We could also start to think about how we would implement reading from a source file using the data models you've created in your projects.

Reviewer: We can delete this part because we will answer their questions asynchronously and the class time is in fact just the bi-weekly showcase if I have understood correctly.

```suggestion
## Project preparation
Create a giter8 template project: https://github.com/scala/scala-seed.g8.
In this project create a basic implementation of what we have in Ingestion Factory with a Feed that has an Origin and Destination trait defined.
```
